app/models/public_json.rb

```diff
@@ -8,4 +8,8 @@ def data=(data)
   def data
     ActiveSupport::Gzip.decompress(super)
   end
+
+  def cocina_hash
+    JSON.parse(data)
```

Comment:
Memoize?

Reply:
Unnecessary. The caller is memoizing the result.